dotcom-rendering/src/web/layouts/FrontLayout.tsx

```diff
@@ -34,26 +35,91 @@ const spaces = / /g;
 const ophanComponentId = (name: string) =>
 	name.toLowerCase().replace(spaces, '-');
 
-const shouldInsertMerchHigh = (
-	index: number,
-	isNetworkFront: boolean | undefined,
+const getMerchHighPosition = (
 	collectionCount: number,
-	isPaidContent: boolean | undefined,
+	isNetworkFront: boolean | undefined,
 ) => {
-	const minContainers = isPaidContent ? 1 : 2;
-	if (collectionCount < minContainers) {
-		return false;
-	}
-	let desiredPosition;
 	if (collectionCount < 4) {
-		desiredPosition = 2;
+		return 2;
 	} else if (isNetworkFront) {
-		desiredPosition = 5;
+		return 5;
 	} else {
-		desiredPosition = 4;
+		return 4;
 	}
+};
+
+// On mobile, we remove the first container if it is a thrasher
+// and remove a container if it, or the next sibling, is a commercial container
+// we also exclude any containers that are directly before a thrasher
+// then we take every other container, up to a maximum of 10, for targeting MPU insertion
+
+const getMobileAdPositions = (
```

Comment:
If you turn this into JSDoc, it can help with documentation.

```suggestion
/**
  * On mobile, we remove the first container if it is a thrasher
  * and remove a container if it, or the next sibling, is a commercial container
  * we also exclude any containers that are directly before a thrasher
  * then we take every other container, up to a maximum of 10, for targeting MPU insertion
  */
const getMobileAdPositions = (
```

Reply:
+1 to using JSDoc here, the integration with VSCode and other IDEs is great